integration-tests/src/test/java/io/druid/tests/indexer/ITKafkaIndexingServiceTest.java

```diff
@@ -184,16 +181,22 @@ public void testKafka()
         throw Throwables.propagate(ioe);
       }
 
-      try {
-        Thread.sleep(DELAY_BETWEEN_EVENTS_SECS * 1000);
-      }
-      catch (InterruptedException ex) { /* nothing */ }
       dtLast = dt;
       dt = new DateTime(zone);
     }
 
     producer.close();
 
+    LOG.info("Waiting for [%s] millis for Kafka indexing tasks to consume events", WAIT_TIME_MILIIS);
```

Comment:
Would be nicer to wait _up to_ `WAIT_TIME_MILLIS` so if the tasks load faster than that, the test can exit earlier. But, this is fine for now, as ITs are not expected to be the fastest things in the world.